cli/util/cancel.go

```diff
@@ -72,6 +76,30 @@ func cancelTx(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to send conflicting transaction: %w", err), 1)
 	}
-	fmt.Fprintln(ctx.App.Writer, resHash.StringLE())
+	var (
+		acceptedH = resHash
+		res       *state.AppExecResult
+	)
+	if ctx.Bool("await") {
+		res, err := a.WaitAny(gctx, resVub, txHash, resHash)
+		if err != nil {
+			if errors.Is(err, waiter.ErrTxNotAccepted) {
+				if mainTx == nil {
+					fmt.Fprintf(ctx.App.Writer, "Neither target nor conflicting transaction is accepted before the current height %d (ValidUntilBlock value of conlicting transaction). Main transaction is unknown to the provided RPC node, thus still has chances to be accepted, you may try cancellation again\n", resVub)
+					return nil
```

Comment:
OK, this one is a pure error, should return this text as an error, because transaction cancellation failed in fact.